src/networks.json

```diff
@@ -875,6 +890,23 @@
     "start": 53370,
     "imageIPFS": "QmPXPCBho3kGLt5rhG9JGkKmzdtLvqZmJqGzzijVCuggWY"
   },
+  "5851": {
+    "key": "5851",
+    "name": "Ontology Testnet",
+    "chainId": 5851,
+    "network": "testnet",
+    "multicall": "0x381445710b5e73d34aF196c53A3D5cDa58EDBf7A",
+    "rpc": [
+      "https://polaris1.ont.io:10339",
+      "https://polaris2.ont.io:10339",
+      "https://polaris3.ont.io:10339",
+      "https://polaris4.ont.io:10339",
+      "https://polaris4.ont.io:10339",
+      "https://polaris5.ont.io:10339"
```

Comment:
All these nodes don't work? only the first one works perfectly.

Reply:
Could you please tell me which method you called?